src/pages/es/guides/typescript.md

```diff
@@ -11,26 +11,15 @@ Astro no realiza ninguna verificación de tipos por sí mismo. La verificación
 
 ## Configuración
 
-Se **recomienda encarecidamente** que crees un archivo `tsconfig.json` para que las herramientas como Astro y VSCode sepan cómo interpretar tu proyecto. Algunas funcionalidades (como las importaciones de paquetes npm) no son totalmente compatibles con TypeScript a menos que crees un archivo `tsconfig.json`.
+[] incluye un archivo `tsconfig.json` en tu proyecto. Incluso si tu no escribes código TypeScript, este archivo es importante pues así, herramientas como Astro y VS Code saben como entender tu proyecto. Algunas características(como importación de archivos npm) no están totalmente soportadas en el editor sin un archivo `tsconfig.json`. Si instalas Astro manualmente, asegurate de create este archivo por tú mismo.
 
-Algunas opciones de configuración de TypeScript requieren atención especial en Astro. A continuación te recomendamos un archivo `tsconfig.json` básico, que puedes copiar y pegar en tu proyecto. Cada [plantilla en astro.new](https://astro.new/) incluye este archivo `tsconfig.json` por defecto.
+Tres plantillas extensibles del archivo `tsconfig.json` están incluidas con Astro: `base`, `strict` y `strictest`. La plantilla `base` habilita el soporte para para características modernas de JavaScript y también es la base de las otras plantillas. Nosotros recomendamos usar las plantillas `strict` o `strictest` si planeas escribir TypeScript en tu proyecto. Puedes ver y comparar las tres plantillas en [astro/tsconfigs/](https://github.com/withastro/astro/blob/main/packages/astro/tsconfigs/).
```

Comment:
```suggestion
Tres plantillas extensibles del archivo `tsconfig.json` están incluidas con Astro: `base`, `strict` y `strictest`. La plantilla `base` habilita el soporte para características modernas de JavaScript y también es usada como base para las otras plantillas. Recomendamos usar las plantillas `strict` o `strictest` si planeas escribir TypeScript en tu proyecto. Puedes ver y comparar las tres plantillas en [astro/tsconfigs/](https://github.com/withastro/astro/blob/main/packages/astro/tsconfigs/).
```

Reply:
Thanks for the recommendations. I will implement the suggested updates.